packages/strapi-tiptap-editor/admin/src/components/extensions/Price/index.ts

```diff
@@ -59,17 +61,15 @@ export const Price = Node.create({
   },
   renderHTML({ HTMLAttributes }) {
     return [
-      'react-widget',
+      'span',
       mergeAttributes(HTMLAttributes, {
-        id: HTMLAttributes.id,
+        'data-strapi-idref': HTMLAttributes['data-strapi-idref'],
```

Comment:
Mist hier `data-strapi-category`?

Reply:
Ik denk dat we deze attribuut niet opnieuw hoeven toe te voegen, aangezien we deze al bovenaan hebben toegevoegd in de functie `addAttributes`, waar we de waarde 'price' als standaard hebben gebruikt.

https://github.com/frameless/strapi/commit/c4a2ad4b5746e92da0fcef086ad4acd3b21a7bba#diff-ca175bab47ce9143f932bde29b4b4a95228d4058ddba8f9f6141eb779b8cd747R38